contracts/finance/andromeda-splitter/src/contract.rs

```diff
@@ -271,6 +284,49 @@ fn execute_update_lock(ctx: ExecuteContext, lock_time: Expiry) -> Result<Respons
     ]))
 }
 
+fn execute_default_recipient(
```

Comment:
Same as above. 

I wonder if it'd be worth moving these functions to a shared mod in `andromeda-finance`? Wdyt?

Reply:
How would we handle `SPLITTER.load(deps.storage)?;` and `SPLITTER.save(deps.storage, &splitter)?;` Wouldn't importing these values create a circular dependency?

Reply:
We'd have to do a fair bit of refactoring. Maybe not worth the effort. Was mostly thinking of future code maintenance.